### PR TITLE
Enable trigger autogenerate docs workflow

### PR DESCRIPTION
**Pull request by r3nic1e**

### CODE REVIEW

File: .github/workflows/notify.yml [hunk 1]
```diff
@@ -8,18 +8,42 @@ jobs:
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get tag
+        id: vars
+        run: echo ::set-output name=tag::${GITHUB_REF#refs/*/}
+
+      - name: Trigger helmwave/docs workflow
+        uses: actions/github-script@v5
+        with:
+          github-token: ${{ secrets.GH_DOCS_TOKEN }}
```
Comment (r3nic1e) [inline .github/workflows/notify.yml:37]: @zhilyaev I need your help with setting this secret (it should be PAT for helmwave/docs)

Comment (zhilyaev) [reply .github/workflows/notify.yml:37]: use `${{ secrets.PAT }}`

**Review by zhilyaev — APPROVED** [review]

File: .github/workflows/notify.yml (hunk 1 above)
Comment (zhilyaev) [reply .github/workflows/notify.yml:37]: What are the required permissions?

Comment (r3nic1e) [reply .github/workflows/notify.yml:37]: @zhilyaev it should be `actions:write` - https://docs.github.com/en/rest/reference/actions#create-a-workflow-dispatch-event

**Review by zhilyaev — APPROVED** [review]

**Pull request merged**